files/build_templates/docker_image_ctl.j2

```diff
@@ -113,7 +113,10 @@ function preStartAction()
     fi
 
     echo -n "" > /tmp/snmpagentaddr.yml
-    keys=`/usr/bin/redis-cli -n 4 keys "SNMP_AGENT_ADDRESS_CONFIG|*"`
+    # we should avoid using 'keys' operation via redis-cli or sonic-db-cli
```

Comment:
add explicit TODO as a marker in the comment.

Reply:
updated the TODO marker, it trigger the testing again and you may need to approve it later again.